ci/make_container_images.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script is intended to be used by Cirrus-CI, from the VM
+# built by the 'image_builder' makefile target in this repo.  Use
+# of this script in any other context/environment is unlikely to
+# function as intended.
+
+set -e
+
+SCRIPT_FILEPATH=$(realpath "$0")
+SCRIPT_DIRPATH=$(dirname "$SCRIPT_FILEPATH")
+REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
+
+# shellcheck source=./lib.sh
+source "$REPO_DIRPATH/lib.sh"
+
+# When running under Cirrus-CI on a branch, also tag/push "latest" images
+PUSH_LATEST=0
+
+# shellcheck disable=SC2154
+if [[ -z "$CI" ]] || [[ "$CI" != "true" ]] || [[ "$CIRRUS_CI" != "$CI" ]]; then
+    die "Unexpected \$CI=$CI and/or \$CIRRUS_CI=$CIRRUS_CI"
```

Comment:
A good habit when displaying variables is to show them in quotes:
```
   ... \$CI='$CI' and/or ...
```
This way if the variable is empty, or has a space, or an equals, or just generally is weird, the value is clearer to the reader.

Reply:
oh yes, I usually try to do that.  Good catch.